### DISCUSSION TITLE
Where to open Issues on Projects (ner_conll03)

### DISCUSSION THREAD

**Discussion by maximilianreimer** [explosion/spaCy#10036] [Help: Coding & Implementations] [answered]
Hello I would like to open an Issue / ask a question about the project [ner_conll03](https://github.com/explosion/projects/tree/v3/benchmarks/ner_conll03). Is there a place to do that? 

Specifically, I would like to know where to get the  conll2003 assets needed to run this project.

**Comment by polm** [comment, marked as answer]
It's fine to ask here.

The [CoNLL2003 homepage](https://www.clips.uantwerpen.be/conll2003/ner/) has information on how to get the data. You need to pay for a license from various organizations.
